hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/ProcessWatcher.java

```diff
@@ -0,0 +1,35 @@
+package org.apache.mesos.hdfs.executor;
```

Comment:
love this!  the processWatch and Handler should belong in a non HDFS package space (to be moved to commons later)  the point is they have nothing to do with hdfs or the executor.  They are used by them.

on the other hand the ProcessFailureExitHandler is an HDFS thing and should probably be named such.